projects/core/src/product/connectors/reviews/product-reviews.adapter.ts

```diff
@@ -0,0 +1,10 @@
+import { Observable } from 'rxjs';
+import { Review } from '../../../occ/occ-models/occ.models';
+
+export abstract class ProductReviewsAdapter {
+  abstract loadList(
```

Comment:
let's add a line of doc for abstract class methods

Reply:
👍